catkin_tools/verbs/catkin_clean/cli.py

```diff
@@ -191,21 +193,17 @@ def clean_profile(opts, profile):
 
     # Initialize action options
     if clean_all:
-        opts.logs = opts.build = opts.devel = opts.install = True
+        opts.spaces = [k for k in Context.SPACES.keys() if k != 'source']
 
-    # Make sure the user intends to clena everything
-    spaces_to_clean = (opts.logs or opts.build or opts.devel or opts.install)
+    # Make sure the user intends to clean everything
     spaces_to_clean_msgs = []
 
-    if spaces_to_clean and not (opts.yes or opts.dry_run):
-        if opts.logs and logs_exists:
-            spaces_to_clean_msgs.append(clr("[clean] Log Space:     @{yf}{}").format(ctx.log_space_abs))
-        if opts.build and build_exists:
-            spaces_to_clean_msgs.append(clr("[clean] Build Space:   @{yf}{}").format(ctx.build_space_abs))
-        if opts.devel and devel_exists:
-            spaces_to_clean_msgs.append(clr("[clean] Devel Space:   @{yf}{}").format(ctx.devel_space_abs))
-        if opts.install and install_exists:
-            spaces_to_clean_msgs.append(clr("[clean] Install Space: @{yf}{}").format(install_path))
+    if opts.spaces and not (opts.yes or opts.dry_run):
+        for space in opts.spaces:
+            if getattr(ctx, '{}_space_exists'.format(space))():
+                space_name = Context.SPACES[space]['space']
+                space_abs = getattr(ctx, '{}_space_abs'.format(space))
+                spaces_to_clean_msgs.append(clr("[clean] {}:     @{yf}{}").format(space_name, space_abs))
```

Comment:
```suggestion
                spaces_to_clean_msgs.append(clr("[clean] {:14} @{yf}{}").format(space_name + ':', space_abs))
```
I would suggest this change to align the output (as it was the case before).